### PR TITLE
Add a PKGBUILD for building on Arch Linux

### PR DESCRIPTION
**Pull request by LupertEverett**
It builds on my machine:tm: :V

Ideally this should be on AUR but it probably won't hurt to have it here too.

(Please do tell if I'm missing something here)

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+build() {
+  cd SurrealEngine
+  [[ -d build ]] && rm -rf build
+  mkdir build && cd build
+  cmake -DCMAKE_BUILD_TYPE=Release ..
+
+  make -j 16
+}
+
+package() {
```
Comment (sharkwouter) [inline PKGBUILD:27]: Don't forget to include the relevant license files.

**Comment by sharkwouter** [comment]
Hey @LupertEverett I took a quick look. I know I'm not a big contributor to this project, but I have some experience with packaging. I hope my review is appreciated. I added some comments on what I think could be added to make the package better, but I do think it is a good looking PKGBUILD already.

File: PKGBUILD (hunk 1 above)
Comment (LupertEverett) [reply PKGBUILD:32]: Now the LICENSE and README gets included in the package too! Thanks for the heads up!

File: PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Note that you'll need to supply the UE1 games you want to play yourself
+# And need a Vulkan capable GPU to run SurrealEngine
+
+pkgname=surrealengine-git
+pkgdesc="A reimplementation of Unreal Engine 1 (Git version)"
+pkgver=0.0.1
```
[the remark this answers is not in the document]
Comment (LupertEverett) [reply PKGBUILD:6]: Added! Though since we don't have any tags yet I opted for the revision number option.

**Comment by LupertEverett** [comment]
> Hey @LupertEverett I took a quick look. I know I'm not a big contributor to this project, but I have some experience with packaging. I hope my review is appreciated. I added some comments on what I think could be added to make the package better, but I do think it is a good looking PKGBUILD already.

Thanks for your suggestions! This is literally the first time I've dabbled with PKGBUILDs (honestly all I did here was checking how others made em lol) so any help is appreciated!

File: PKGBUILD (hunk 2 above)
[the remark this answers is not in the document]
Comment (sharkwouter) [reply PKGBUILD:6]: Great, looks good now :)

**Review by sharkwouter — APPROVED** [review]
Looks great now, good work

**Comment by dpjudas** [comment]
OK merging it then. Thanks for looking into it sharkwouter. :)

**Comment by FabioLolix** [comment]
Hello, I've seen the pkgbuild on AUR, it need/could have some changes

**Comment by LupertEverett** [comment]
> Hello, I've seen the pkgbuild on AUR, it need/could have some changes

Hello there!

Could you describe what needs to be changed in either here or in the AUR page? I'll try to look into it ASAP!

**Comment by FabioLolix** [comment]
> Could you describe what needs to be changed in either here or in the AUR page? I'll try to look into it ASAP!

I'll make directly a PR :)

**Pull request merged**